frontend/src/views/camp/Dashboard.vue

```diff
@@ -1,121 +1,157 @@
-                              dateShort(scheduleEntry.end)
-                                ? ''
-                                : dateShort(scheduleEntry.start)
-                            }}
-                            <b> {{ hourShort(scheduleEntry.start) }} </b>
-                            -
-                            {{
-                              dateShort(scheduleEntry.start) ==
-                              dateShort(scheduleEntry.end)
-                                ? ''
-                                : dateShort(scheduleEntry.end)
-                            }}
-                            <b> {{ hourShort(scheduleEntry.end) }} </b>
-                          </v-card-subtitle>
-                        </v-card>
-                      </v-col>
-                    </template>
-                  </template>
-                </template>
-              </v-row>
-            </v-expansion-panel-content>
-          </v-expansion-panel>
-        </template>
-      </v-expansion-panels>
-    </template>
-    <template v-else>
-      <v-card-text>
-        <p style="text-align: center">
-          {{ $tc('views.camp.dashboard.noActivitiesLine1') }} <br />
-          {{ $tc('views.camp.dashboard.noActivitiesLine2') }} <br />
-        </p>
-        <p style="text-align: center">
-          <v-btn color="success" :to="campRoute(camp(), 'program')">
-            <v-icon size="150%" left> mdi-view-dashboard-variant </v-icon>
-            {{ $tc('views.camp.dashboard.program') }}
-          </v-btn>
-        </p>
-      </v-card-text>
-    </template>
+  <content-card :title="$tc('views.camp.dashboard.activities')" toolbar>
+    <div class="d-flow-root">
+      <div
+        v-if="!loading"
+        class="d-flex flex-wrap ma-4"
+        style="overflow-y: auto; gap: 10px"
+      >
+        <BooleanFilter
+          v-model="showOnlyMyActivities"
+          :label="$tc('views.camp.dashboard.onlyMyActivities')"
+        />
+        <FilterDivider />
+        <SelectFilter
+          v-model="filter.responsible"
+          multiple
+          :items="campCollaborations"
+          :display-field="campCollaborationDisplayName"
+          :label="$tc('views.camp.dashboard.responsible')"
+        >
+          <template #item="{ item }">
+            <UserAvatar
+              :camp-collaboration="campCollaborations[item.value]"
+              size="18"
+              class="mr-1"
+            />
+            {{ item.text }}
+          </template>
+        </SelectFilter>
+        <SelectFilter
+          v-model="filter.category"
+          multiple
+          :items="categories"
+          display-field="short"
+          :label="$tc('views.camp.dashboard.category')"
+        >
+          <template #item="{ item }">
+            <CategoryChip dense :category="categories[item.value]" class="mr-1" />
+            {{ categories[item.value].name }}
+          </template>
+        </SelectFilter>
+        <SelectFilter
```

Comment:
Period filter is not shown to me:

<img width="275" alt="image" src="https://user-images.githubusercontent.com/449555/200751932-a3dfca2f-1603-48a9-b539-44a952155b20.png">

Reply:
That is deliberate. It's only showing as soon as there are at least two periods with activities in them. Before that, the period filter does not make sense.

Reply:
Ah, now I understand: I created a 2nd period but had no scheduleEntry planned within this period. I kind of would have expected, that the filter would still show up in that case but not returning any results when selected.

Speaking of no results, some kind of fallback-text would be nice:
"No activities found. Try clearing the selection filters and/or reloading the page."

Reply:
Good point. It's more consistent the way you propose. Suppose I had 3 periods, 1 of which has no schedule entries, I would also expect to see the period filter, and expect to see all 3 periods selectable in it.